frontends/web/src/routes/account/manage/manage.tsx

```diff
@@ -108,7 +129,6 @@ class ManageAccount extends Component<Props, State> {
         this.setState((state) => {
             switch (state.step) {
                 case 0: return ({ step: 1 });
-                case 1: return ({ step: 2 });
```

Comment:
why did you remove this?

Reply:
hm just realized it is outdated... 🤔

Reply:
It happens when the apiPost above succeeds. If adding the account fails, we shouldn't show the success message.

Reply:
yep makes sense